app/src/main/res/layout/item_susi_map.xml

```diff
@@ -30,7 +30,7 @@
 
                 <TextView
                     android:id="@+id/text"
-                    android:layout_width="wrap_content"
+                    android:layout_width="240dp"
```

Comment:
Then `match_parent` should do it. The problem with hardcoding is that if we change one of the dimens, we will have to change all and that will be difficult to track